### PR TITLE
feat: add <ConfirmationDialog>

### PR DESCRIPTION
**Pull request by emmenko**
Based on #367 

Refs #323

screenshot 2019-02-18 at 21 23 28

### CODE REVIEW

File: packages/application-components/src/components/dialogs/confirmation-dialog/README.md [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+| `title`                         | `string`                   |    ✅    | -        | -                                 | The title of the Info Dialog                                                                                                                                                                                                    |
+| `isOpen`                        | `boolean`                  |    ✅    | -        | -                                 | Indicates whether the dialog is open or closed. The parent component needs to manage this state                                                                                                                                 |
+| `onClose`                       | `function`                 |    ✅    | -        | -                                 | Called when the dialog closes (click on overlay, click on close button, press ESC)                                                                                                                                              |
```
Comment (montezume) [inline packages/application-components/src/components/dialogs/confirmation-dialog/README.md]: click on confirm button?

File: packages/application-components/src/components/dialogs/confirmation-dialog/README.md [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+### Static properties
+
+#### `ConfirmationDialog.Intl`
+
+Exposes common intl messages to be used for the secondary/primary buttons
+
+```js
+ConfirmationDialog.Intl.cancel;
+ConfirmationDialog.Intl.confirm;
+```
```
Comment (qmateub) [inline packages/application-components/src/components/dialogs/confirmation-dialog/README.md]: This is nice 👌

File: packages/application-components/src/components/dialogs/internals/dialog-styles.mod.css [hunk 3]
```diff
@@ -47,10 +47,12 @@
 .dialog-content {
   border-top: 1px solid var(--border-color-separator);
   padding-top: var(--spacing-16);
-  max-height: 80vh;
+  max-height: 75vh;
   overflow: auto;
 }
 
+/* TODO: remove the styles below once `Spacings.Inputs` supports `justifyContent` prop */
```
Comment (qmateub) [inline packages/application-components/src/components/dialogs/internals/dialog-styles.mod.css]: 🔝

**Comment by emmenko** [comment]
Alright, this should also be good to go now. The last change I did was basically to make the modal "not closable" by not passing the `onClose` prop.

**Pull request merged**